client/js/color.js

```diff
@@ -0,0 +1,57 @@
+export function toHex(inputColor) {
+  const hexPattern = /^#([A-Fa-f0-9]{6})$/;
+  if (hexPattern.test(inputColor)) {    
+    return inputColor;
+  }
+  const canvas = document.createElement('canvas');
+  const ctx = canvas.getContext('2d');
+  ctx.fillStyle = inputColor;
+  const hexColor = ctx.fillStyle;  
+  canvas.remove();
+  return hexColor;
+}
+
+export function toRGB(inputColor) {
+  let hex = toHex(inputColor)
+  hex = hex.slice(1);
+  var rgb = hex.match(new RegExp('(.{' + hex.length/3 + '})', 'g')).map(function(l) {
+    return parseInt(hex.length%2 ? l+l : l, 16);
```

Comment:
I beleive that toHex() is returning starndardized #rrggbb values, so test of length is not needed.

Reply:
Good point.  That was leftover from when each function did its own thing before going through toHex.  Hopefully this is better.